### PR TITLE
test: additional jest tests for tasks

### PR DESCRIPTION
**Pull request by karel-rehor**
Adds more jest test for tasks to improve code coverage

### CODE REVIEW

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 59 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=1724)

File: src/tasks/containers/TasksPage.test.tsx [hunk 1]
```diff
@@ -180,4 +239,103 @@ describe('Tasks.Containers.TasksPage', () => {
...
+    })
+
+    /*
+    NB was working up until most recent rebase 0bfc0bb370af 21.6.16
+    Now throws error on fire event
+
+    [Util] handleError::  Cannot assign to read only property 'status' of object '#<Object>'
+     :-/
+    */
+    it.skip('activates a task', async () => {
```
Comment (genehynson) [inline src/tasks/containers/TasksPage.test.tsx:320]: @karel-rehor I think this error started after we updated the `immer` library recently. It became stricter about modifying component props directly. I was able to fix the issue in [this commit](https://github.com/influxdata/ui/commit/200af228869f7759d6ef5452681237edb8680328). I don't have permission to modify this fork so I'll leave it to you to bring those changes into this PR

**Review by genehynson — COMMENTED** [review]
nice! Bring in the changes I linked to fix that one skipped test and then this PR should be good to go

File: src/tasks/components/TaskCard.test.tsx [hunk 2]
```diff
@@ -58,19 +57,5 @@ describe('Tasks.Components.TaskCard', () => {
...
-      const {ui, props} = setup()
-
-      fireEvent.click(ui.getByTestId('task-card--slide-toggle'))
-      expect(props.onActivate.mock.calls[0][0].status).toEqual('inactive')
-
-      fireEvent.click(ui.getByTestId('task-card--slide-toggle'))
-      expect(props.onActivate.mock.calls[1][0].status).toEqual('active')
-    })
-  })
+  // N.B. moved activation test to TasksPage container test
```
Comment (genehynson) [inline src/tasks/components/TaskCard.test.tsx:60]: acknowledged. I think you can remove this comment

**Review by genehynson — APPROVED** [review]
looks great, thanks

**Pull request merged**